docs/changelog-fragments.d/194.bugfix.md

```diff
@@ -0,0 +1,3 @@
+Ensure ANSI escapes are removed from stdout/stderr
+[#373][https://github.com/ansible/vscode-ansible/issues/373]
```

Comment:
Mind using a detached link for this? It's usually easier to read plain-text markdown with these things separted.